list.go

```diff
@@ -150,6 +150,85 @@ func (db *RoseDB) LIndex(key []byte, index int) ([]byte, error) {
 	return val, nil
 }
 
+// LRange returns the specified elements of the list stored at key.
+// The offsets start and stop are zero-based indexes,
+// with 0 being the first element of the list (the head of the list),
+// 1 being the next element and so on.
+// These offsets can also be negative numbers indicating offsets starting at the end of the list.
+// For example, -1 is the last element of the list, -2 the penultimate, and so on.
+// If start is larger than the end of the list, an empty list is returned.
+// If stop is larger than the actual end of the list, Redis will treat it like the last element of the list.
```

Comment:
I think you can shorten the comment lines. For example, line 154, and 156 is a little bit short.

Reply:
yes, sounds good, fixed